butterknife-sample/src/test/java/com/example/butterknife/unbinder/UnbinderTest.java

```diff
@@ -16,22 +16,22 @@
 import static com.google.common.truth.Truth.assertThat;
 
 @RunWith(RobolectricTestRunner.class) //
-@Config(manifest = "src/main/AndroidManifest.xml")
+@Config(sdk = {21}, manifest = "src/main/AndroidManifest.xml")
```

Comment:
@JakeWharton Yeah. 

@ChristianBecker If you use `roboletric.properties`(see https://github.com/jaredsburrows/android-gradle-java-app-template/blob/master/src/test/resources/robolectric.properties), you do not need to specify this here. Also, `manifest` should automatically be picked up(See https://github.com/jaredsburrows/android-gradle-java-app-template/blob/master/src/test/java/test/RoboTestBase.java#L21).